metadata.yaml

```diff
@@ -22,11 +22,17 @@ containers:
     mounts:
       - storage: database
         location: /var/lib/prometheus
+
 provides:
+  self-metrics-endpoint:
+    interface: prometheus_scrape
   grafana-source:
     interface: grafana_datasource
+  grafana-dashboard:
```

Comment:
Should this be
```suggestion
  self-grafana-dashboard:
```
?

Reply:
I'm not sure.... @simskij what do you think?